packages/headless/src/app/case-assist-engine/case-assist-engine.ts

```diff
@@ -77,7 +77,11 @@ export function buildCaseAssistEngine(
     reducers: caseassistEngineReducers,
   };
 
-  const engine = buildEngine(augmentedOptions, thunkArguments);
+  const engine = buildEngine(
+    augmentedOptions,
+    thunkArguments,
+    caseAssistAPIClient
```

Comment:
Why pass a third argument instead of accessing the client from `thunkArguments`?